### PR TITLE
Add missing DOT OAuth2 config for ecommerce.

### PR DESCRIPTION
**Pull request by doctoryes**
The conversion of ecommerce from using DOP to DOT during the last hackathon left a loose end - ecommerce auth has been broken since then for sandboxes. This PR adds the necessary configuration to enable OAuth SSO auth between ecommerce and the LMS on a newly-built sandbox.

I've created a sandbox using this branch and tested auth, which works. If you'd like to try it, it's here:
https://julia-ecom6.sandbox.edx.org/

and here's the Jenkins provisioning which created it:
https://tools-edx-jenkins.edx.org/job/sandboxes/job/CreateSandbox/21105/

---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Review by nasthagiri — APPROVED** [review]
Thanks for finding and fixing this @doctoryes!

File: playbooks/roles/ecommerce/defaults/main.yml [hunk 1]
```diff
@@ -280,7 +285,7 @@ ecommerce_redhat_pkgs: []
...
+  - command: './manage.py create_or_update_site --site-id=1 --site-domain={{ ECOMMERCE_ECOMMERCE_URL_ROOT.split("://")[1] }} --partner-code=edX --partner-name="Open edX" --lms-url-root={{ ECOMMERCE_LMS_URL_ROOT }} --client-side-payment-processor=cybersource --payment-processors=cybersource,paypal --client-id={{ ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_KEY }} --client-secret={{ ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_SECRET }} --sso-client-id={{ ECOMMERCE_SOCIAL_AUTH_EDX_OAUTH2_KEY }} --sso-client-secret={{ ECOMMERCE_SOCIAL_AUTH_EDX_OAUTH2_SECRET }} --backend-service-client-id={{ ECOMMERCE_BACKEND_SERVICE_EDX_OAUTH2_KEY }} --backend-service-client-secret={{ ECOMMERCE_BACKEND_SERVICE_EDX_OAUTH2_SECRET }} --from-email staff@example.com --discovery_api_url={{ ECOMMERCE_DISCOVERY_SERVICE_URL }}/api/v1/ --journals_api_url={{ JOURNALS_API_URL|default("") }}'
```
Comment (pwnage101) [inline playbooks/roles/ecommerce/defaults/main.yml:288]: Is `--client-id` and `--client-secret` necessary anymore?

**Review by pwnage101 — APPROVED** [review]
Other than it seeming to be an expand-only change (rather than expand/contract, removing the old id and secret settings), LGTM

**Comment by doctoryes** [comment]
@pwnage101 Good point on expansion without contraction. However, to avoid any risk of synchronization problems between configuration versus other repos/builds, I'm going to keep this PR expansion-only at the moment. Eventual DOP removal will force these contraction changes.

**Pull request merged**